packages/migrate/test/migrate.test.ts

```diff
@@ -166,6 +166,39 @@ export default class Hello extends Component {
       expectFile(outputs[0]).toEqual(expected);
     });
 
+    test('with service', async () => {
```

Comment:
@cafreeman I know the task is to deal with namespaced services but can you add a test case for non namespaced services to make sure what we get as output is sane?